javascript/javascript1/week2/get-students.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+function addStudentToClass(studentName){
+    if(class07Students.length ===6 && studentName!=="Queen")
+       { return console.log(`Can not  add  more students to class 07`);
```

Comment:
You have a bracket that has escaped from the line above :-) Keep a consistent style.